cranecloud/docs/authentication/registration.md

```diff
@@ -0,0 +1,40 @@
+## Registration
+1\. To register visit cranecloud.io and click the `GET STARTED `Button
```

Comment:
link changed to <http://staging.cranecloud.io/login>

Reply:
This url too has been changed, @MutegekiHenry